extra/modules/confiant-ad-quality/src/test/java/org/prebid/server/hooks/modules/com/confiant/adquality/v1/ConfiantAdQualityBidResponsesScanHookTest.java

```diff
@@ -218,6 +218,60 @@ public void shouldSubmitToScanBidsWhichAreNotPartOfTheExcludeToScanConfig() {
         )));
     }
 
+    @Test
+    public void shouldSubmitToScanOnlyBidsWithData() {
+        // given
+        final String secureBidderName = "securebidder";
+        final String notSecureBadBidderName = "notsecurebadbidder";
+        final String emptyBidderName = "emptybidder";
+        final BidderResponse secureBidderResponse = AdQualityModuleTestUtils.getBidderResponse(secureBidderName, "imp_a", "bid_id_a");
+        final BidderResponse notSecureBadBidderResponse = AdQualityModuleTestUtils.getBidderResponse(notSecureBadBidderName, "imp_b", "bid_id_b");
+        final BidderResponse emptyBidderResponse = AdQualityModuleTestUtils.getEmptyBidderResponse(emptyBidderName);
+
+        final ConfiantAdQualityBidResponsesScanHook hookWithExcludeConfig = new ConfiantAdQualityBidResponsesScanHook(
```

Comment:
just override the `hook` and describe the condition of excluding the config in the test name

Reply:
Sorry @AntoxaAntoxic that need to disturb you again. But I didn't get our aim.
I'm setting a another hook instance inside @Before phase.
Will it work for us if I extend it with a separate hook?
```
    @Before
    public void setUp() {
        hook = new ConfiantAdQualityBidResponsesScanHook(bidsScanner, List.of(), privacyEnforcementService);
        hookWithExcludedBiddersFromScan = new ConfiantAdQualityBidResponsesScanHook(bidsScanner, List.of(SECURE_BIDDER), privacyEnforcementService);
    }
```

Reply:
I was talking about code style again and naming of the so-called `service under testing` naming, usually we use the word `target` (you can check richmedia module code). In your case you're using `hookWithExcludeConfig` that is better to rename to just `hook` to be aligned with other tests or even better rename everything to the `target`

Reply:
please check the code style here docs/developers/code-style.md for more info

Reply:
Hope I got you right. Decided to update tests names related to the class which I touched.
Thanks.